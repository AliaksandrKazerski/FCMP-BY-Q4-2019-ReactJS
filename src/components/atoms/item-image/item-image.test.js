import React from 'react';
import { shallow } from 'enzyme';

import ItemImage from './index';

describe('ItemImage', () => {
  it("ItemImage should render correctly", () => {
  const myComponent = shallow(<ItemImage/>);
  
  expect(myComponent).toMatchSnapshot()
  });
});