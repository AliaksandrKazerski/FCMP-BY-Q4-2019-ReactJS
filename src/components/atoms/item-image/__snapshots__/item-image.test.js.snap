// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemImage ItemImage should render correctly 1`] = `ShallowWrapper {}`;
